.changeset/tall-jars-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/dev": major
```

Comment:
Since Remix supports Node@>=18.0.0 and this is available as of Node v17, this is not a breaking change

```suggestion
"@remix-run/dev": patch
```